### PR TITLE
Add Devolutions Workspace version 2024.1.1.0

### PR DESCRIPTION
**Pull request by devolutionsbot**
This is an automated pull request to add Workspace version 2024.1.1.0
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/144158)

### CODE REVIEW

File: manifests/d/Devolutions/Workspace/2024.1.1.0/Devolutions.Workspace.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Devolutions.Workspace
+PackageVersion: 2024.1.1.0
```
Comment (OfficialEsco) [inline manifests/d/Devolutions/Workspace/2024.1.1.0/Devolutions.Workspace.installer.yaml:4]:
```suggestion
PackageVersion: 24.1.1.0
```

File: manifests/d/Devolutions/Workspace/2024.1.1.0/Devolutions.Workspace.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Devolutions.Workspace
+PackageVersion: 2024.1.1.0
```
Comment (OfficialEsco) [inline manifests/d/Devolutions/Workspace/2024.1.1.0/Devolutions.Workspace.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 24.1.1.0
```

File: manifests/d/Devolutions/Workspace/2024.1.1.0/Devolutions.Workspace.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Devolutions.Workspace
+PackageVersion: 2024.1.1.0
```
Comment (OfficialEsco) [inline manifests/d/Devolutions/Workspace/2024.1.1.0/Devolutions.Workspace.yaml:4]:
```suggestion
PackageVersion: 24.1.1.0
```

**Review by OfficialEsco — COMMENTED** [review]
The PackageVersion should match the DisplayVersion, however i see #141335 got merged so it should also be fixed..

**Comment by rbstp** [comment]
> The PackageVersion should match the DisplayVersion, however i see #141335 got merged so it should also be fixed..

The version on our website is the same as the package version that was submitted: https://devolutions.net/workspace/

image

**Comment by stephengillie** [comment]
Verify: 
image

Maybe a `DisplayVersion` might be appropriate here?

**Comment by rbstp** [comment]
This was added in the Display version, as suggested. I have also notified the developers, they will fix it in the next release so the version is the same as the package version.

File: manifests/d/Devolutions/Workspace/2024.1.1.0/Devolutions.Workspace.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+UpgradeBehavior: install
+Protocols:
+- credential
+- http
+- https
+ProductCode: '{A4BCFDF0-CBDC-4622-83C4-EB17E6C98CE2}'
+ReleaseDate: 2024-03-13
+AppsAndFeaturesEntries:
+- UpgradeCode: '{EA58A259-5C39-43E8-9183-EAAB09293D3F}'
+- DisplayVersion: 24.1.1
```
Comment (stephengillie) [inline manifests/d/Devolutions/Workspace/2024.1.1.0/Devolutions.Workspace.installer.yaml:23]:
```suggestion
- DisplayVersion: 24.1.1.0
```
Sorry, but I believe the DisplayVersion has to be an exact match, including the trailing zero.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**